src/clusterfuzz/_internal/cron/triage.py

```diff
@@ -98,6 +98,54 @@ def _is_bug_filed(testcase):
   return False
 
 
+def _is_startup_crash(testcase):
+  """Check if job contains both libfuzzer and android"""
```

Comment:
"Checks"
Also, it should be documented in this function name or at least docstring that this only applies to android/libFuzzer. I could only tell from the PR name which isn't going to be available when reading the code.

Reply:
Done.